Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -130,12 +130,19 @@ class SentryUserFeedbackForm: UIViewController {
             let list = missing.count == 1 ? missing[0] : missing[0 ..< missing.count - 1].joined(separator: ", ") + " and " + missing[missing.count - 1]
             let alert = UIAlertController(title: "Error", message: "You must provide all required information. Please check the following field\(missing.count > 1 ? "s" : ""): \(list).", preferredStyle: .alert)
             alert.addAction(UIAlertAction(title: "OK", style: .default))
-            present(alert, animated: config.animations)
+            present(alert, animated: config.animations) {
+                if let block = self.config.onSubmitError {
+                    block(NSError(domain: "io.sentry.error", code: 1, userInfo: ["missing_fields": missing, NSLocalizedDescriptionKey: "The user did not complete the feedback form."]))
```

Comment:
`m`: When I change this to the following, no test fails. It would be great to have a test asserting the contents of NSError.
```suggestion
                    block(NSError(domain: "io.sentry", code: 1, userInfo: ["missing_fields": missing, NSLocalizedDescriptionKey: "The user did not complete the feedback form."]))
```

Also a test should fail when the code doesn't invoice the block, which isn't the case right now. The same applies to `onSubmitSuccess`.

Reply:
When I comment out the invocation of `onSubmitSuccess`, `testSubmitWithOnlyRequiredFieldsFilled`, `testSubmitFullyFilledForm` and `testSubmissionErrorThenSuccessAfterFixingIssues` fail.

For `onSubmitError`, the following fail: 
testSubmissionErrorThenSuccessAfterFixingIssues()
testSubmitOnlyWithOptionalFieldsFilled()
testSubmitWithNoFieldsFilledAllRequired()
testSubmitWithNoFieldsFilledAllRequiredCustomLabels()
testSubmitWithNoFieldsFilledDefault()
testSubmitWithNoFieldsFilledEmailAndMessageRequired()

What did you try that you didn't see the failures?

Reply:
Maybe I got it wrong. I'm sorry, I can't recall. If you're confident tests are failing, I trust you. Thanks for double checking.